habitat-hitl/habitat_hitl/_internal/hitl_driver.py

```diff
@@ -520,6 +521,30 @@ def depth_to_rgb(obs):
             self._remote_gui_input.on_frame_end()
 
         if self.network_server_enabled:
+            if (
+                self._hitl_config.networking.active_features.camera_sync
+                and "cam_transform" in post_sim_update_dict
+            ):
+                cam_transform: Optional[mn.Matrix4] = post_sim_update_dict[
+                    "cam_transform"
+                ]
+                if cam_transform is not None:
+                    cam_position = cam_transform.translation
+                    cam_rotation = mn.Quaternion.from_matrix(
+                        cam_transform.rotation()
+                    )
+                    if self._client_message_manager:
```

Comment:
Why check `self.network_server_enabled` AND `self._client_message_manager`? Why do any work at all here if you're not going to send a message?